debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/events/Transaction.java

```diff
@@ -102,7 +113,7 @@ public int hashCode() {
     public String toString() {
         return "Transaction{" +
                 "transactionId='" + transactionId + '\'' +
-                ", startScn=" + startScn +
-                '}';
+                ", startScn=" + startScn + ", userName='" + userName +
```

Comment:
Could you move the `userName` portion to the next line to follow the same formatting?